app/ide-desktop/lib/client/electron-builder-config.ts

```diff
@@ -92,14 +92,16 @@ export const args: Arguments = await yargs(process.argv.slice(2))
 // ======================================
 
 /** Based on the given arguments, creates a configuration for the Electron Builder. */
-export function createElectronBuilderConfig(passedArgs: Arguments): electronBuilder.Configuration {
+export async function createElectronBuilderConfig(
+    passedArgs: Arguments
+): Promise<electronBuilder.Configuration> {
     return {
         appId: 'org.enso',
         productName: common.PRODUCT_NAME,
         extraMetadata: {
             version: BUILD_INFO.version,
         },
-        copyright: 'Copyright © 2022 ${author}.',
+        copyright: `Copyright © ${new Date().getFullYear()} $\{author}.`,
```

Comment:
not sure why the `{` is now escaped - i'm pretty sure it'll make this text show up as it is: `${author}`

Reply:
Ugh, it was auto-converted, as previously the string wasn't interpolated at all.
Still, I'm going to address this as part of this PR..